app/src/main/java/com/duckduckgo/app/browser/defaultbrowsing/prompts/ui/DefaultBrowserBottomSheetDialog.kt

```diff
@@ -47,8 +47,8 @@ class DefaultBrowserBottomSheetDialog(private val context: Context) : BottomShee
             setRoundCorners(dialogInterface)
             eventListener?.onShown()
         }
-        setOnDismissListener {
-            eventListener?.onDismissed()
+        setOnCancelListener {
```

Comment:
what’s the reasoning for this?

Reply:
Explained in https://github.com/duckduckgo/Android/pull/5543#discussion_r1942559065.